grails-app/domain/com/careerfly/user/Profile.groovy

```diff
@@ -0,0 +1,22 @@
+package com.careerfly.user
+
+import com.careerfly.organisation.College
+import com.careerfly.organisation.School
+import com.careerfly.taggable.Tag
+
+class Profile {
+
+    User user
+    Set skills
+    Set interests
+    static hasMany = [skills : Tag, interests : Tag]
```

Comment:
Move this line above `constraints` so that `static` blocks are in group. This is just for code readability. @akshitagangrade @ankit18singh @apoorvsrivastava1116 @Smita007 @atul12594 @shubhamj15 @Sagars09